src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -130,6 +130,45 @@
 
   </Target>
 
+  <!--
+    Temporary until we have proper nuget support to deploy content files.
+    Copies supplemental test data to the build output and test directories.
+  -->
+  <Target Name="CopySupplementalTestData"
+          AfterTargets="CopyTestToTestDirectory"
+          Inputs="@(SupplementalTestData)"
```

Comment:
It would be defined by the projects that require it.  See https://github.com/jhendrixMSFT/corefx/commit/adf0c5927bce1ed7ec97f56577719d63049f24a6 for how this will be consumed.